package.json

```
@@ -6,18 +6,20 @@
   "scripts": {
     "build": "sass scss/bitstyles.scss build/bitstyles.css --style=compressed --no-source-map",
     "postbuild": "postcss build/bitstyles.css --replace",
-    "lint:scss": "stylelint scss/**/*.scss --syntax scss",
-    "lint:js": "eslint --ignore-path .gitignore --ext js,jsx .",
-    "lint": "yarn lint:scss && yarn lint:js",
+    "lint:scss": "stylelint --ignore-path .gitignore 'scss/**/*.scss' --customSyntax postcss-scss && yarn prettier:scss",
+    "lint:js": "eslint --ignore-path .gitignore --ext js,jsx . && yarn prettier:js",
+    "lint:mdx": "prettier --ignore-path .gitignore --check './**/*.mdx'",
```

Hm, I guess we're missing the `.md` files with that and you already ran prettier on them. Ignoring them from now on could mean they go off the rails.

How about:

```suggestion
    "lint:docs": "prettier --ignore-path .gitignore --check './**/*.{md,mdx}'",
```

---

👍 done (needed another change where this task was used in another task)